cluster/kubemark/iks/config-default.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Cloud information
+RANDGEN=$(dd if=/dev/urandom bs=64 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=16 count=1 2>/dev/null | sed 's/[A-Z]//g')
+KUBE_NAMESPACE="kubemark_${RANDGEN}"
+KUBEMARK_IMAGE_TAG="${KUBEMARK_IMAGE_TAG:-2}"
```

Comment:
Note that hardcoding the image tag to a constant value may not be the best idea (as you may overwrite it each time you run the script).

Reply:
The image tag is set there to be the initial value for the correct kubemark repo image, assuming the user does not want to build the image. However, I can change it if necessary.